controllers/topics-controllers

```diff
@@ -0,0 +1,12 @@
+const {
```

Comment:
This is not a js file. Don't forget to add `.js` at the end of the file or it will not work. This suggest you likely did not run your tests as it would have failed with a broken controller like this.